<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xToolsBleCentralToolUi</class>
 <widget class="QWidget" name="xToolsBleCentralToolUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>166</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Devices</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="xToolsBleDeviceInfoComboBox" name="comboBoxDevices"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonSettings">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonScan">
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Service</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="comboBoxServices">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Characteristic</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QComboBox" name="comboBoxCharacteristics"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelWriteWay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Write way</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QComboBox" name="comboBoxWriteWay">
     <item>
      <property name="text">
       <string>WriteWithResponse</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WriteWithoutResponse</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="labelUnsupported">
     <property name="text">
      <string>(Unsupported characteristic)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonNotify">
       <property name="text">
        <string>Notify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRead">
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xToolsBleDeviceInfoComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsBleDeviceInfoComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
